ZHMModSDK/Include/Events.h

```diff
@@ -3,8 +3,11 @@
 #include "EventDispatcher.h"
 #include "Common.h"
 
+#include <vector>
+#include <string>
+
 class ZHMSDK_API Events
 {
 public:
-    static EventDispatcher<void>* OnConsoleCommand;
+    static EventDispatcher<std::vector<std::string>>* OnConsoleCommand;
```

Comment:
Should not use standard library constructs across the SDK API boundary since slight changes in it (eg. in new compiler versions) can break ABI compatibility. Use the Z* and T* classes instead (e.g. `ZString` and `TArray`, though `TArray` might be incomplete atm).